src/coprocessor/dag/expr/ctx.rs

```diff
@@ -34,8 +34,24 @@ pub const FLAG_OVERFLOW_AS_WARNING: u64 = 1 << 6;
 // FLAG_DIVIDED_BY_ZERO_AS_WARNING indicates if DividedByZero should be returned as warning.
 pub const FLAG_DIVIDED_BY_ZERO_AS_WARNING: u64 = 1 << 8;
 
-pub const MODE_NO_ZERO_DATE_MODE: u64 = 25;
-pub const MODE_ERROR_FOR_DIVISION_BY_ZERO: u64 = 27;
+bitflags! {
+    /// Please refer to SQLMode in `mysql/const.go` in repo `pingcap/parser` for details.
+    pub struct SqlMode: u64 {
+        const STRICT_TRANS_TABLES = 1 << 22;
+        const STRICT_ALL_TABLES = 1 << 23;
+        const NO_ZERO_IN_DATE = 1 << 24;
+        const NO_ZERO_DATE = 1 << 25;
+        const INVALID_DATES = 1 << 26;
```

Comment:
Is this necessary for this PR?

Reply:
yes, since we are now using values from sql_mode (but which is wrong)